tutorials/SpectroscopicDataReductionBasics/3-Trace_Extract_Wavelength-CalibrateSpectrum.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
<p>gone through Spectroscopic Data Reduction 1 and 2 on finding the trace and wavelength calibration.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>